docker-compose.yml

```diff
@@ -345,11 +347,66 @@ services:
       - _APP_DB_USER
       - _APP_DB_PASS
       - _APP_FUNCTIONS_TIMEOUT
+      - _APP_FUNCTIONS_BUILD_TIMEOUT
       - _APP_FUNCTIONS_CONTAINERS
       - _APP_FUNCTIONS_RUNTIMES
       - _APP_FUNCTIONS_CPUS
       - _APP_FUNCTIONS_MEMORY
       - _APP_FUNCTIONS_MEMORY_SWAP
+      - _APP_EXECUTOR_SECRET
+      - _APP_USAGE_STATS
+      - DOCKERHUB_PULL_USERNAME
+      - DOCKERHUB_PULL_PASSWORD
+
+  appwrite-executor:
+    container_name: appwrite-executor
+    entrypoint:
+      - php
+      - -e
+      - /usr/src/code/app/executor.php
+      - -dopcache.preload=opcache.preload=/usr/src/code/app/preload.php
```

Comment:
We should create an executable file on our Dockerfile, like we do for other services, same for the `compose.phtml` file.

Reply:
Done 👍